content-learn/mastering-meshery-istio/introduction-to-service-meshes/linkerd/deploy-an-application.mdx

```diff
@@ -0,0 +1,195 @@
+---
+docType: "Chapter"
+chapterTitle: "Deploy a sample application"
+description: "Meshery is the service mesh managemen plane which offers lifecycle, configuration, and performance management of service meshes and their workloads."
+videos: 4
+lectures: 12
+---
+
+import { ChapterStyle } from "../../../../src/components/Learn-Components/Chapters-Style/chapters.style.js";
+
+<ChapterStyle>
+
+To play with Linkerd and demonstrate some of it's capabilities, you will deploy the sample application, EmojiVoto.
+
+<h2 className="chapter-sub-heading">What is the EmojiVoto Application </h2>
+<br />
+
+Emojivoto is a sample microservice application that allows users to vote for their favorite emoji.
+It displays votes received on a leaderboard. Emojivoto has no dependecies on Linkerd, but will run 
+fine either on or off the service mesh.
+
+A sidecar injector is used for automating the injection of the Linkerd proxy into your application's pod spec. The Kubernetes admission controller enforces this behavior send 
+sending a webhook request the the sidecar injector every time a pod is to be scheduled. This injector inspects resources for a Linkerd-specific annotation (linkerd.io/inject: enabled). 
+When that annotation exists, the injector mutates the pod's specification and adds both an init container as well as a sidecar containing the proxy itself.
+
+The Linkerd sidecar proxy can be either manually or automatically injected into your application's pods. This can be done from the meshery dashboard for namespace globally by,
+
+1. Using Meshery, navigate to the Linkerd management page. 
+1. Click the (+) icon on the `Configure Application` card and select `Annotate Namespace` from the list.
+
+As part of Linkerd deployment in [Previous chapter](/learn-ng/mastering-meshery-istio/introduction-to-service-meshes/linkerd/getting-started), you have deployed the sidecar injector.
+To verify, execute this command:
+
+```sh
+kubectl get deployment linkerd-proxy-injector -n linkerd
+```
+
+Output:
+
+```sh
+NAME                     READY   UP-TO-DATE   AVAILABLE   AGE
+linkerd-proxy-injector   1/1     1            1           9m49s
+```
+
+Examine the annotation added to the `linkerd` namespace:
+
+```sh
+kubectl describe namespace linkerd
+```
+
+Output:
+
+```sh
+Name:         linkerd
+Labels:       config.linkerd.io/admission-webhooks=disabled
+              linkerd.io/control-plane-ns=linkerd
+              linkerd.io/is-control-plane=true
+Annotations:  linkerd.io/inject: disabled
+Status:       Active
+
+No resource quota.
+
+No LimitRange resource.
+```
+
+<h3 class="chapter-sub-heading">
+  <a name="auto"></a> Deploy the application
```

Comment:
my bad, I just copied it. 😓

Reply:
after removing `a` tag.
Are we good to go?

Reply:
Ok, it might be needed in one of the future chapter. But, let's not add now. If at all needed, ca be added in the future while migrating other chapters.